MD/TemporalCore/MetadataMDTemporalCore.rdf

```diff
@@ -22,14 +22,14 @@
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/MD/TemporalCore/MetadataMDTemporalCore/">
 		<rdfs:label>Metadata for the EDMC-FIBO Market Data (MD) TemporalCore Module</rdfs:label>
 		<dct:abstract>This is the metadata ontology used to describe the MD Temporal Core Module.</dct:abstract>
```

Comment:
Not duplicating the abstract from the Module?

Reply:
fixed.